tests/Backend/CommonPart1/BucketsTest.php

```diff
@@ -60,23 +60,51 @@ public function testBucketsList(string $devBranchType, string $userRole): void
 
         $this->assertTrue(count($buckets) >= 2);
 
+        $firstBucket = null;
         $inBucketFound = false;
         $outBucketFound = false;
         foreach ($buckets as $bucket) {
             if ($bucket['id'] == $this->getTestBucketId(self::STAGE_IN)) {
                 $inBucketFound = true;
+                $firstBucket = $bucket;
             }
             if ($bucket['id'] == $this->getTestBucketId(self::STAGE_OUT)) {
                 $outBucketFound = true;
             }
         }
         $this->assertTrue($inBucketFound);
         $this->assertTrue($outBucketFound);
+        $this->assertNotNull($firstBucket);
 
-        $firstBucket = reset($buckets);
+        $this->assertArrayHasKey('id', $firstBucket);
+        $this->assertSame($this->getTestBucketId(self::STAGE_IN), $firstBucket['id']);
+        $this->assertArrayHasKey('name', $firstBucket);
         $this->assertArrayHasKey('displayName', $firstBucket);
         $this->assertNotEquals('', $firstBucket['displayName']);
         $this->assertArrayHasKey('created', $firstBucket);
+        $this->assertArrayHasKey('uri', $firstBucket);
```

Comment:
přidal sem nějaké asserty, ale ty responses sů dost málo otestované jako takové takže sem to dal do risku, minule byl incident na komponent detail https://keboola.atlassian.net/browse/RISK-64